src/Diagnostics/Samples/CSharp/CSharpAnalyzers/CSharpAnalyzers/DiagnosticIds.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace CSharpAnalyzers
+{
+    public static class DiagnosticIds
+    {
+        // Stateless analyzer IDs.
+        public const string SymbolAnalyzerRuleId = "CSharpSampleAnalyzer0001";
```

Comment:
We should promote short errorcodes otherwise the error list's errorcode column starts looking ugly. I suggest CSS001